tests/test_schema_validator.py

```diff
@@ -1,7 +1,7 @@
+import sys
 import os
-
+sys.path.append(os.getcwd())
```

Comment:
Do we need these? (besides `import os`)

Reply:
Receive the following error when running tests/test_schema_validator.py with
```
import os
from xmltodict import parse
from pyQuARC.code.schema_validator import SchemaValidator

```
```
Traceback (most recent call last):
  File "/Users/sam/Documents/GitHub/pyQuARC/tests/test_schema_validator.py", line 3, in <module>
    from pyQuARC.code.schema_validator import SchemaValidator
ModuleNotFoundError: No module named 'pyQuARC'
```